### ISSUE TITLE
Announce screen changes to screen readers

### ISSUE THREAD

**Issue by distantorigin** [hariettem/Artemis#279]
**Is your accessibility request related to a problem? Please describe.**
Currently, activating many items does not announce that the screen has changed to screen readers. This includes, but is not limited to:

* Activating a post from the Posts tab
* Activating an item in the Inbox tab
* Navigating into any sub-category in settings (i.e. Appearance).
* Probably others (will add in comments as I discover them)

**Describe the solution you'd like**
Announce changes to the screen when controls are activated or actions are taken that effect the view. Also announce when changes to focus occur.

This library has some convenience functions for doing this: https://www.npmjs.com/package/react-native-a11y

**Describe alternatives you've considered**

**Additional context**

**Comment by hariettem** [comment]
Thanks for the library link! 👀
